packages/payment-backend/src/enums.ts

```diff
@@ -9,3 +9,13 @@ export enum PaymentType {
   CREDIT_CARDS = 'creditcards',
   CARD_RENEWAL = 'payment_card_renewal',
 }
+
+export enum ReferenceType {
+  ORDER = 'order',
+  MERCHANT = 'merchant',
+}
+
+export enum FilterType {
```

Comment:
filterType (group tai code) This is duplicate of ReferenceType, you should read story comments in jira to get this right.

Reply:
`{
    "name": "Nordea",
    "code": "nordea",
    "group": "banks",
    "img": "https://www.vismapay.com/e-payments/method_logos/nordea.png"
  },
  {
    "name": "Visa",
    "code": "creditcards",
    "group": "creditcards",
    "img": "https://www.vismapay.com/e-payments/method_logos/visa.png"
  },
  {
    "name": "Pivo",
    "code": "pivo",
    "group": "wallets",
    "img": "https://www.vismapay.com/e-payments/method_logos/pivo.png"
  }
  `
That is the json that needs to be filtered,
  
ReferenceType tells us if it is merchant or order, but in this case it is order. 
FilterType can be `"banks" or "creditcards" or "wallets"`
Value references "code" field in json.

Reply:
I was thinking about to remove this FilterType enum because this can change when paytrail is used.

Reply:
I did not notice the comments on the ticket. Makes sense now and I agree with the removal of FilterType as it seems useless in this case. I'll remove and change it to a regular string.